back/spec/acceptance/static_pages_spec.rb

```diff
@@ -195,6 +229,18 @@
         assert_status 200
         expect { page.reload }.to raise_error ActiveRecord::RecordNotFound
       end
+
+      context 'when the page has a code other than \'custom\'' do
+        before do
+          page.code = 'faq'
+          page.save!
```

Comment:
(just a note, up to you) You can also use a shorter version `before { page.update!(code: 'faq') }`